.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  # For GitHub Actions workflows, update all actions on the main branch
+  - package-ecosystem: "github-actions"
+    groups:
+      actions on main branch:
+        patterns:
+          - "*"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "scalar-labs/scalardb"
```

Comment:
The whole team uses this repository, but I set the DB team as reviewers for starters.
Let me know if I need to set somebody else.